common-rendering/src/components/keyEvents.tsx

```diff
@@ -0,0 +1,162 @@
+import { FC, useState } from "react";
+import { textSans, headline } from "@guardian/src-foundations/typography";
+import { palette, remSpace } from "@guardian/src-foundations";
+import { SvgChevronUpSingle, SvgChevronDownSingle } from "@guardian/src-icons";
+import { Theme } from "@guardian/types";
+import { from } from "@guardian/src-foundations/mq";
+import { getColor, KeyEvent } from "../keyEvent";
+
+// ----- Component ----- //
+
+const keyEventWrapperStyles: SerializedStyles = css`
+	width: 100%;
+
+	${from.desktop} {
+		width: 220px;
+	}
+`;
+
+const titleRowStyles: SerializedStyles = css`
+	position: relative;
+	margin: ${remSpace[1]} ${remSpace[3]} 0;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
+	svg {
+		height: 30px;
+	}
+
+	path {
+		fill: ${palette.neutral[46]};
+	}
+
+	${from.phablet} {
+		margin: ${remSpace[1]} ${remSpace[5]} 0;
+	}
+
+	${from.desktop} {
+		display: none;
+	}
+`;
+
+const titleStyle: SerializedStyles = css`
+	${headline.xxsmall({ fontWeight: "bold", lineHeight: "tight" })};
+	color: ${palette.neutral[7]};
+`;
+
+const buttonStyles: SerializedStyles = css`
+	border: 0;
+	background: 0;
+	outline: 0;
+	padding: 0;
+`;
+
+const listStyles: SerializedStyles = css`
+	margin: ${remSpace[3]};
+
+	${from.phablet} {
+		margin: ${remSpace[3]} ${remSpace[5]} 0;
+	}
+
+	${from.desktop} {
+		margin: ${remSpace[1]} 0 0;
+	}
+
+	li::before {
+		content: "";
+		border-color: transparent ${palette.neutral[7]};
+		border-style: solid;
+		border-width: 0.4rem 0 0.4rem 0.5rem;
+		display: block;
+		height: 0;
+		width: 0;
+		top: 0;
+		position: absolute;
+	}
+`;
+
+const listItemStyles: SerializedStyles = css`
+	padding-bottom: ${remSpace[3]};
+	border-left: 1px solid ${palette.neutral[7]};
+	position: relative;
+
+	&:last-child {
+		border-left: none;
+	}
+`;
+
+const timeTextWrapperStyles: SerializedStyles = css`
+	margin-left: ${remSpace[4]};
+`;
+
+const textStyles = (theme: Theme): SerializedStyles => css`
+	${headline.xxxsmall({ fontWeight: "regular", lineHeight: "regular" })};
+	color: ${getColor(theme, 300)};
+	text-decoration: none;
+`;
+
+const timeStyles: SerializedStyles = css`
+	${textSans.xxsmall({ fontWeight: "bold", lineHeight: "tight" })};
+	color: ${palette.neutral[7]};
+`;
+
+interface KeyEventsProps {
+	keyEvents: KeyEvent[];
+	theme: Theme;
+}
+
+interface ListItemProps {
+	keyEvent: KeyEvent;
+	theme: Theme;
+}
+
+const ListItem: FC<ListItemProps> = ({ keyEvent, theme }) => {
+	return (
+		<li css={listItemStyles}>
+			<div css={timeTextWrapperStyles}>
+				<div css={timeStyles}>{keyEvent.time}</div>
+				<a css={textStyles(theme)} href={keyEvent.url}>
+					{keyEvent.text}
+				</a>
+			</div>
+		</li>
+	);
+};
+
+const KeyEvents: FC<KeyEventsProps> = ({ keyEvents, theme }) => {
+	const [showList, setShowList] = useState(true);
+
+	return (
+		<div css={keyEventWrapperStyles}>
+			<div css={titleRowStyles}>
+				<div css={titleStyle}>Key Events</div>
+				<button
```

Comment:
If we use [Button from source](https://guardian.github.io/source/?path=/story/source-src-button-button--playground) we get lots of really nice accessibility wins

Reply:
Thanks @oliverlloyd  When I use the source button, it's adding some hover styles, Is it possible/do we need to remove this ?

Reply:
I just added screenshots for when it's hovered

Reply:
I really like @JamieB-gu 's idea to use [details](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details). 

But to your question about hover. In the past, [I've also had issues](https://github.com/guardian/gateway/blob/7c89002de1c8fc2cb5d06672c72f3b31b6f1081a/src/client/components/Footer.tsx#L30) with Source's hover styles and I used `cssOverrides` to remove them. I find it's good to start with too much  styling (by using Source) and remove or override it as needed, hopefully this way we're less likely to forget to implement the accessibility features that get baked in to Source.

Reply:
Thanks both. I'll work on using summary/details